ethapi/backend.go

```diff
@@ -76,7 +78,26 @@ type Backend interface {
 	GetEvent(ctx context.Context, shortEventID string) (*inter.Event, error)
 	GetEventHeader(ctx context.Context, shortEventID string) (*inter.EventHeaderData, error)
 	GetConsensusTime(ctx context.Context, shortEventID string) (inter.Timestamp, error)
-	GetHeads(ctx context.Context, epoch int) (hash.Events, error)
+	GetHeads(ctx context.Context, epoch rpc.BlockNumber) (hash.Events, error)
```

Comment:
add comment about the following scenarios:
 
epoch=-1, 
0<= epoch <= current epoch
epoch > current epoch

Reply:
The same mechanics as for web3 methods, like getBlockByNumber

Reply:
Added comments